config/webpack.config.prod.js

```diff
@@ -193,6 +194,18 @@ module.exports = {
     new MiniCssExtractPlugin({
       filename: 'main.css',
     }),
+    new BundleAnalyzerPlugin(
+      {
+        analyzerMode: 'static',
+        reportFilename: 'report.html',
+        defaultSizes: 'parsed',
+        openAnalyzer: true,
+        generateStatsFile: false,
+        statsFilename: 'stats.json',
```

Comment:
As `generateStatsFile` is `false` it wont create a file name `stats.json`

Reply:
Removed.